lib/wsdl.js

```diff
@@ -1206,7 +1206,98 @@ WSDL.prototype.toXML = function() {
   return this.xml || '';
 };
 
-WSDL.prototype.xmlToObject = function(xml) {
+WSDL.prototype.findInNamespace = function (splitPrefix_Name) {
```

Comment:
Please don't use `_` in variables, we use camelCase notation so please rename `splitPrefix_Name` to `splitPrefixName`